### PR TITLE
[5.2] Uppercase and not-exported README files

### PR DESCRIPTION
**Pull request by lucasmichot**
- Ensure all `.md` files are uppercase
- Do not export `README.md`
- Do not export `README.md` for `Illuminate/Database`

### CODE REVIEW

File: src/Illuminate/Database/.gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.gitattributes export-ignore
+README.md export-ignore
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/.gitattributes]: other places have a readme too

Comment (lucasmichot) [inline src/Illuminate/Database/.gitattributes]: Added `src/Illuminate/Queue/.gitattributes`

**Comment by GrahamCampbell** [comment]
I don't think this entirely necessary tbh.

**Comment by GrahamCampbell** [comment]
:-1: from me

**Pull request closed without merging**